CONTRIBUTING.adoc

```diff
@@ -550,6 +551,32 @@ vector: 'logos/${imageId}/${imageId}.svg'
 credit: 'Your Name'
 credit_url: 'https://twitter.com/yourtwitteraccount'
 ```
+=== Adding a book
+
+When creating a submission for the link:/books/[Jenkins Books] page, prepare a `.yml` file with the following configuration:
+
+````yaml
+----
```

Comment:
```suggestion
```yaml
---
```

Three will make it valid. Same for the closing backticks at line 570.

Reply:
Have fixed this in the commit I have pushed just now as well.